plugins/ua_pubsub_ethernet.c

```diff
@@ -814,7 +818,8 @@ UA_PubSubChannelEthernet_open(const UA_PubSubConnectionConfig *connectionConfig)
         }
     }
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,19,0)
+#if defined(KERNEL_VERSION)
```

Comment:
Please combine the two #if into one.
Just concatenation the expressions with &&.